databases/spatialite/Portfile

```diff
@@ -4,6 +4,7 @@ PortSystem          1.0
 
 name                spatialite
 version             4.99.0
+revision            1
```

Comment:
The `revision` should be set back to 0. Adding a variant does not require a `revision` change.